packages/camera/camera_android_camerax/example/lib/camera_controller.dart

```diff
@@ -526,7 +526,7 @@ class CameraController extends ValueNotifier<CameraValue> {
     }
 
     if (value.isStreamingImages) {
-      stopImageStream();
+      await stopImageStream();
```

Comment:
Adding the `await` here will make it take longer to actually begin the process of stopping recording on the native side; do we want that? An alternative would be to store the future in a local variable, and then await it just before the `return`.

Reply:
I don't think we want this...I'll remove the await.